pkg/dataobj/internal/dataset/column.go

```diff
@@ -30,22 +41,22 @@ type (
 // portion of the column at a time.
 type Column interface {
 	// ColumnInfo returns the metadata for the Column.
-	ColumnInfo() *ColumnInfo
+	ColumnInfo() *ColumnDesc
```

Comment:
Update method name to ColumnDesc?

PageInfo should also be updated to PageDesc to align